index.html

```diff
@@ -23,6 +23,7 @@ <h1>IMA SDK Plugin for Video.js</h1>
 <p><a href="examples/advanced/">View the Advanced Example</a></p>
 <p><a href="examples/playlist/">View the Playlist Example</a></p>
 <p><a href="examples/autoplay/">View the Autoplay Example</a></p>
+<p><a href="examples/multiple/">View the Multiple Example</a></p>
```

Comment:
Multiple players? This is a little unclear.

Reply:
it won't have that nice symmetry, but done.